src/ttauri/pixel_map.hpp

```diff
@@ -211,6 +211,7 @@ class pixel_map {
 
     pixel_map &operator=(pixel_map &&other) noexcept
     {
+        // Self assignment is allowed.
```

Comment:
Remove the comment below it. Which meant the same

Reply:
Addressed in d2100bc455c4e29e86b1a744defb8229b1de469e